agenta-web/src/components/pages/evaluations/NewEvaluation/SelectVariantSection.tsx

```diff
@@ -0,0 +1,187 @@
+        {
+            title: "Name",
+            dataIndex: "variant_name",
+            key: "variant_name",
+            fixed: "left",
+            onHeaderCell: () => ({
+                style: {minWidth: 160},
+            }),
+            render: (_, record) => {
+                return <span>{record.variantName}</span>
+            },
+        },
+        {
+            title: "Last modified",
+            dataIndex: "updatedAt",
+            key: "updatedAt",
+            onHeaderCell: () => ({
+                style: {minWidth: 160},
+            }),
+            render: (_, record) => {
+                return <div>{record.updatedAt}</div>
+            },
+        },
+    ]
+
+    if (isDemo()) {
+        columns.push({
+            title: "Modified by",
+            dataIndex: "modifiedById",
+            key: "modifiedById",
+            onHeaderCell: () => ({
+                style: {minWidth: 160},
+            }),
+            render: (_, record) => {
+                return <div>{usernames[record.modifiedById]}</div>
+            },
+        })
+    }
+
+    columns.push(
+        // {
+        //     title: "Tags",
+        //     onHeaderCell: () => ({
+        //         style: {minWidth: 160},
+        //     }),
+        // },
+        {
+            title: "Model",
+            dataIndex: "parameters",
+            key: "model",
+            onHeaderCell: () => ({
+                style: {minWidth: 160},
+            }),
+            render: (_, record) => {
+                return record.parameters && Object.keys(record.parameters).length
+                    ? Object.values(
+                          filterVariantParameters({record: record.parameters, key: "model"}),
+                      ).map((value, index) => (value ? <Tag key={index}>{value}</Tag> : "-"))
+                    : "-"
+            },
+        },
+        {
+            title: "Created on",
+            dataIndex: "createdAt",
+            key: "createdAt",
+            onHeaderCell: () => ({
+                style: {minWidth: 160},
+            }),
+            render: (_, record) => {
+                return <div>{record.createdAt}</div>
+            },
+        },
+    )
+
+    const filteredVariant = useMemo(() => {
+        if (!searchTerm) return variants
+        return variants.filter((item) =>
+            item.variantName.toLowerCase().includes(searchTerm.toLowerCase()),
+        )
+    }, [searchTerm, variants])
+
+    const selectedVariants = useMemo(
+        () => variants.filter((variant) => selectedVariantIds.includes(variant.variantId)),
+        [variants, selectedVariantIds],
+    )
+
+    const handleRemoveVariant = (variantId: string) => {
+        const filterVariant = selectedVariantIds.filter((id) => variantId !== id)
+        setSelectedVariantIds(filterVariant)
+    }
+
+    return (
+        <Collapse
+            defaultActiveKey={["1"]}
+            {...props}
+            items={[
```

Comment:
let's improve this part. This is not good practice, because these'll be recreated each render. even if they don't change React will see the entire items prop as a new object and triggers unnecessary re-renders.